.rubocop_todo.yml

```diff
@@ -13,6 +13,8 @@ Metrics/AbcSize:
 # Offense count: 45
 # Configuration parameters: CountComments, ExcludedMethods.
 Metrics/BlockLength:
+  Exclude:
+    - 'spec/**/*.rb'
```

Comment:
Out of curiosity, what's the reasoning here?

Reply:
Ah, this is because rubocop was complaining about the length of blocks in a test.  In my mind, that isn't really an issue.   I guess you could consider it a code smell if you have too many tests in one rspec test (because the object is too complicated), but 1. I don't think too many tests is an issue for this codebase for a while and 2. I think that complication smell is better detected at the class under test level, which rubocop provides.

Reply:
👍 thought that was the case